source/Camera/Maintenance/CorrectCameraTrueness/CorrectCameraTrueness.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+This example shows how to perform In-field correction. This involves gathering data from
+a compatible calibration object at several distances, calculating an updated camera
```

Comment:
Can you call it 'Zivid in-field calibration object' here as well?

Reply:
Will do.